R/BoxCox.R

```diff
@@ -33,9 +33,14 @@
 #'
 #' # Tidying
 #'
-#' When you [`tidy()`][tidy.recipe()] this step, a tibble with columns
-#' `terms` (the selectors or variables selected) and `value` (the
-#' lambda estimate) is returned.
+#' When you [`tidy()`][tidy.recipe()] this step, a tibble is returned with 3
+#' columns `terms`, `value` , and `id`:
```

Comment:
Looks like you didn't feel on board for removing the number of columns in the description. Grammatically, if I see "is returned with 3 columns", I expect a colon, i.e.

> a tibble is returned with 3 columns:  `terms`, `value` , and `id`.

with second parentheses removed, or 

> a tibble is returned with columns  `terms`, `value` , and `id`:

I recommended the second option because I liked the flow of leading into `\describe{}` with a parentheses, but the first option works, too.

Reply:
done in [dont document number of columns](https://github.com/tidymodels/recipes/pull/1230/commits/48a2b4fd801a2b27482ed529357494c5e8c8be51)